msisensor2/0.1-deb/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM biocontainers/biocontainers:v1.2.0_cv2
+MAINTAINER biocontainers <biodocker@gmail.com>
+LABEL software="msisensor2" \ 
+    container="msisensor2" \ 
+    about.summary="MSIsensor2 is a novel algorithm based machine learning, featuring a large upgrade in the microsatellite instability (MSI) detection for tumor only sequencing data, including Cell-Free DNA (cfDNA), Formalin-Fixed Paraffin-Embedded(FFPE) and other sample types" \
+    about.home="https://github.com/niu-lab/msisensor2" \ 
+    software.version="0.1-deb" \ 
```

Comment:
Why version 0.1-deb?

Should match upstream release (cannot see any info about this)

Reply:
The deb is added for autogenerated continers fromdebian packages

In your case should be version only (version tag and directory)